README.md

```diff
@@ -451,11 +451,15 @@ return_random_cases(ctu05, percent = 0.25, seed = 1337, date = "2019-03-18",
 ```
 
 ```
-##          mnpaid                   centre mnpvisstartdate
-## 2 RPACK-INS-012 Inselspital Bern (RPACK)      2019-04-12
-## 4 RPACK-INS-014 Inselspital Bern (RPACK)      2019-04-14
-## 5 RPACK-CBE-005   Charité Berlin (RPACK)      2019-04-05
-## 3 RPACK-CBE-003   Charité Berlin (RPACK)      2019-04-03
+## $cases
+##           mnpaid                   centre mnpvisstartdate
+## 1  RPACK-INS-011 Inselspital Bern (RPACK)      2019-04-11
+## 4  RPACK-INS-014 Inselspital Bern (RPACK)      2019-04-14
+## 41 RPACK-CBE-004   Charité Berlin (RPACK)      2019-04-04
+## 3  RPACK-CBE-003   Charité Berlin (RPACK)      2019-04-03
+## 
+## $rng_config
+## [1] "Mersenne-Twister" "Inversion"
```

Comment:
More of a note than anything else... you don't have to commit the whole file... you can select just the badge to commit, then this part wont change all the time

Reply:
Would you do that manually or is there a way to commit specific lines?

Reply:
in rstudio you can select which lines to stage. This SO post has a method for the command line https://stackoverflow.com/questions/1085162/commit-only-part-of-a-file-in-git